ssh/DOCS.md

```diff
@@ -6,50 +6,57 @@ Follow these steps to get the add-on installed on your system:
 
 1. This addon is only visible to "Advanced Mode" users. To enable advanced mode, go to **Profile** -> and turn on **Advanced Mode**.
 2. Navigate in your Home Assistant frontend to **Supervisor** -> **Add-on Store**.
-3. Find the "SSH server" add-on and click it.
+3. Find the "Terminal & SSH" add-on and click it.
 4. Click on the "INSTALL" button.
 
-## How to use
-
-You need enable the port for external access. You can just enter 22 as value or any other
-value like you want. This recommend to add login credentials.
-
-To use this add-on, you must have a private/public key to log in.
-To generate them, follow the [instructions for Windows][keygen-windows]
-and [these for other platforms][keygen]. It is possible to set a password for
-login since version 2.0 but for high security use private/public keys.
 
-You can not run both variants at the same time. Enabling login via keys, will
-disable password login.
+## How to use
 
-1. Add a ssh key to  `authorized_keys` or set a `password` in the add-on configuration.
-2. Start the add-on.
-3. Connect to your device using your preferred SSH client and use `root` as
-   the username.
+Once installed (following the steps above) you need to
+* enable external access by selecting which TCP port to open up (see Network section), and
+* supply login credentials,
+* start the add-on.
 
-After logging in, you will find yourself in this add-on’s container.
-The Home Assistant configuration directory is mounted on the path `/config`.
+You can then connect to the port specified, and using the username `root`
+you will end up in this add-on's container. The Home Assistant configuration 
+directory is mounted on the path `/config`.
 
-Try it out using:
+This add-on comes bundled with [The Home Assistant CLI](https://www.home-assistant.io/hassio/commandline/). Try it out using:
 
 ```bash
 ha help
 ```
 
+### General Note on Security
+
+You are strongly recommended to use private/public keys to log in. 
+To generate them, follow the [instructions for Windows][keygen-windows]
+and [these for other platforms][keygen]. It is possible to set a password for
+login since version 2.0 but for high security use private/public keys.
+
+Enabling login via password, will disable password login. You can not run both variants at the same time.
+
+
 ## Configuration
 
 Add-on configuration:
 
 ```yaml
 authorized_keys:
-  - ssh-rsa AKDJD3839...== my-key
+  - "ssh-rsa AKDJD3839...== my-key"
 password: ''
+server:
+  tcp_forwarding: false
 ```
 
 ### Option: `authorized_keys`
 
 Your **public keys** for the authorized key file. You can authorize multiple
-keys by adding multiple public keys to the list.
+keys by adding multiple public keys to the list. If you're on Linux, you can
+likely issue the command `ssh-add -L` to get a list of your public keys that 
```

Comment:
This is unclear to the average user? Do I need to type that command in the terminal provided by the add-on? Or what if I don't have Linux?

Reply:
Thanks, I removed this.